pyaedt/modules/Material.py

```diff
@@ -1992,16 +1993,248 @@ def set_magnetic_coercivity(self, value=0, x=1, y=0, z=0):
         return self.update()
 
     @pyaedt_function_handler()
-    def set_electrical_steel_coreloss(self, kh=0, kc=0, ke=0, kdc=0, cut_depth=0.0001):
-        """Set Electrical Steel Type Core Loss.
+    def get_core_loss_coefficients(
+        self,
+        points_list_at_freq,
+        core_loss_model_type="Electrical Steel",
+        thickness="0.5mm",
+        conductivity=0,
+        coefficient_setup="w_per_cubic_meter",
+    ):
+        """Compute core loss coefficients from the loss characteristics (B-P Curve) at a given frequency.
```

Comment:
```suggestion
        """Compute core loss coefficients from the loss characteristics (BP curve) at a given frequency.
```
Chabge to "BP curve" was to match occurrence in first parameter description. Choose whichever but stay consistent.